changelog_entry.yaml

```
@@ -0,0 +1,4 @@
+bump: {patch}
+  changes:
+    {added}:
+      - readiness_check
```

__issue, blocking:__ This is not formatted correctly
```suggestion
bump: patch
  changes:
    added:
    - readiness_check
```